app/views/provider_interface/interviews/check.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :browser_title, 'Set up an Interview'%>
+<% content_for :before_content, govuk_back_link_to(provider_interface_application_choice_new_reject_path(@application_choice.id)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">
+      <span class="govuk-caption-l"><%= @application_choice.application_form.full_name %></span>
+      <%= t('page_titles.provider.confirm_reject') %>
+    </h1>
+    <%= form_with model: @interview_form, url: new_provider_interface_application_choice_interviews_path(@application_choice.id), method: :post do |f| %>
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <%= f.govuk_error_summary %>
+        </div>
+      </div>
+
+      <%= render ProviderInterface::InterviewDetailsComponent.new(@interview_form) %>
+
+      <%= f.hidden_field 'date(1i)', value: @interview_form.year %>
+      <%= f.hidden_field 'date(2i)', value: @interview_form.month %>
+      <%= f.hidden_field 'date(3i)', value: @interview_form.day %>
+      <%= f.hidden_field :time %>
+      <%= f.hidden_field :location %>
+      <%= f.hidden_field :additional_details %>
+      <%= f.hidden_field :provider_id %>
```

Comment:
Is this the best way to do this?
We might have missed something but this was needed in order to correctly submit from the check details page

Reply:
This doesn't work because the existing params are lost when you click a change link as they don't POST back. We have a standard way of building up params for a check answers page: have  a look at some of the wizards like `ProviderUserInvitationWizard` or `ProviderRelationshipPermissionsSetupWizard` and their collaborating controllers. The `InterviewForm` should be relatively straightforward to convert into a wizard.